### PR TITLE
Add felipecaputo.git-project-manager

### PR DESCRIPTION
**Pull request by 1efty**
Here is the issue asking the maintainer: https://github.com/felipecaputo/git-project-manager/issues/115

Please consider adding this.

### CODE REVIEW

**Review by jankeromnes — APPROVED** [review]
Thanks! While waiting for the maintainer's response, I'm happy to temporarily publish this extension from here.

One small thing though: Why is there a `checkout` parameter? Normally when you provide a VSIX direct `download` URL, there is a `version` but no `checkout`.

Did you add this extension by running `node add-extension https://github.com/felipecaputo/git-project-manager`? (Normally this will auto-detect everything and update `extensions.json` with a correct configuration -- but if it adds a `checkout` here, that's a bug.)

**Comment by 1efty** [comment]
@jankeromnes I added that manually. I've amended the commit using the method you outlined. Appreciate the prompt response.

**Review by jankeromnes — APPROVED** [review]
Very cool, thanks for fixing!

**Pull request merged**
